docs/config.rst

```diff
@@ -163,9 +158,9 @@ Upgrading ``astropy``
 Each time you upgrade to a new major version of ``astropy``, the
 configuration parameters may have changed.
 
-If you never edited your configuration file, there is nothing for you
-to do. It will automatically be replaced with a configuration file
-template for the newly installed version of ``astropy``.
+If you never edited your configuration file, there is nothing for you to do. It
```

Comment:
"there is nothing for you to do" is not entirely accurate anymore? In fact, if users have been relying on `astropy.cfg`, nowt they have to explicitly run `astropy.write_default_config()` to get an updated `astropy.cfg`. Otherwise, the old one stays and might mess things up. Did I understand this new behavior correctly?